File: sendTelegram.js
// where package.json is located: `npm install node-fetch`
// or `npm install -g node-fetch` to install globally
const fetch = require("node-fetch");
const sendTelegram = async (message) => {
	const token = "YOUR_BOT_TOKEN";
	const data = Object.entries({
	    	text: message,
	    	chat_id: "YOUR_CHAT_ID",
	    	parse_mode: "html"
	    })
	    .map(entry => entry[0] + "=" + entry[1])
	    .join("&");

	const url = `https://api.telegram.org/bot${token}/sendMessage?${data}`;
	console.log(`url: ${url}`);
	const response = await fetch(url);
	const json = await response.json();
	console.log(json);
	return json;	
};

sendTelegram("hi!");

File: send_telegram.py
import urllib.parse
import urllib.request

def send_telegram(message):
    token = "YOUR_TOKEN"
    
    data = {
        'text': message,
        'chat_id': 'YOUR_CHAT_ID'
    }
    url = f"https://api.telegram.org/bot{token}/sendMessage?{urllib.parse.urlencode(data)}&parse_mode=html"
    response = urllib.request.urlopen(url)
    return response.read().decode()

send_telegram("hi!")
